pip/vcs/bazaar.py

```diff
@@ -4,16 +4,12 @@
 import os
 import tempfile
 
+from pip._vendor.six.moves.urllib import parse as urllib_parse
```

Comment:
Six does not bring all functions from urlparse to parse.

https://pythonhosted.org/six/#module-six.moves.urllib.parse